config/config.go

```diff
@@ -116,6 +116,7 @@ var (
 	DefaultConsulSDConfig = ConsulSDConfig{
 		TagSeparator: ",",
 		Scheme:       "http",
+		WatchTimeout: 30 * time.Second,
```

Comment:
Any harm in setting this higher, like a couple of minutes? Not that it should matter much, I guess.